src/Outsystems.SetupTools/Functions/Get-OSPerfTuningStatus.ps1

```diff
@@ -0,0 +1,309 @@
+                {
+                    $NETConfig = $(GetDotNetLimits)
+
+                    $CurrentMaxRequestLength = $NETConfig.SystemWeb.HttpRuntime.maxRequestLength
+                    $CurrentMaxRequestLengthInMB = "$($CurrentMaxRequestLength/1024) MB"
+                    $CurrentExecutionTimeout = $NETConfig.SystemWeb.HttpRuntime.executionTimeout
+
+                    # upload size limits default is 4096 KB
+                    #   https://docs.microsoft.com/en-us/dotnet/api/system.web.configuration.httpruntimesection.maxrequestlength
+                    $DefaultMaxRequestLength = 4096
+                    $DefaultMaxRequestLengthInMB = "$($DefaultMaxRequestLength/1024) MB"
+
+                    # execution timeout default is 110 seconds
+                    #   https://docs.microsoft.com/en-us/dotnet/api/system.web.configuration.httpruntimesection.executiontimeout
+                    [TimeSpan]$DefaultExecutionTimeout = "00:01:50"
+
+                    $HasSmashableMaxRequestLength = $CurrentMaxRequestLength -eq $DefaultMaxRequestLength
+                    $HasSmashableExecutionTimeout = $CurrentExecutionTimeout -eq $DefaultExecutionTimeout
+
+                    $ShouldBeSkipped = $(-not $HasSmashableMaxRequestLength) -or $(-not $HasSmashableExecutionTimeout)
+
+                    $CompTextMaxRequestLength = $(GetIsOrIsntText $HasSmashableMaxRequestLength)
+
+                    $Messages += @("Current upload limit value ('$CurrentMaxRequestLengthInMB') $CompTextMaxRequestLength the default ('$DefaultMaxRequestLengthInMB').")
+
+                    $CompTextExecutionTimeout = $(GetIsOrIsntText $HasSmashableExecutionTimeout)
+
+                    $Messages += @("Current execution timeout value ('$CurrentExecutionTimeout') $CompTextExecutionTimeout the default ('$DefaultExecutionTimeout').")
+                }
+                catch
+                {
+                    $Messages = @("Unable to determine current values. Skipping.")
+                }
+
+                return $(CreateResult -ShouldBeSkipped $ShouldBeSkipped -Messages $Messages)
+            }
+
+        $PerfTuning.Sections.IISUploadSizeLimitsConfig = CreatePerfTuningStatus -Title "IIS upload size limits configuration" `
+            -ScriptBlock `
+            {
+                $ShouldBeSkipped = $True
+                $Messages = @()
+
+                try
+                {
+                    $Filter = "system.webServer/security/requestFiltering/requestLimits"
+                    $Name = "maxAllowedContentLength"
+
+                    $MaxAllowedContentLength = $(GetWebConfigurationProperty -PSPath "MACHINE/WEBROOT/APPHOST" -Filter "$Filter" -Name "$Name")
+                    # MaxAllowedContentLength default value is 30000000
+                    #  https://docs.microsoft.com/en-us/iis/configuration/system.webserver/security/requestfiltering/requestlimits/#configuration
+                    $DefaultMaxAllowedContentLength = 30000000
```

Comment:
(same as above) We should move this to a constant in src/Outsystems.SetupTools/Lib/Constants.ps1